src/webgpu/util/compare.ts

```diff
@@ -60,6 +61,12 @@ export function correctlyRoundedMatch(): FloatMatch {
   };
 }
 
+export function intervalMatch(i: F32Interval): FloatMatch {
```

Comment:
Do we still need `FloatMatch` ? Can we not entirely replace `FloatMatch` with the `intervalComparator` ? It would be a worthwhile goal, as it would let us entirely strip out the `cmpFloats` smell.